### PR TITLE
feat: updated ReputationEvent entity

### PR DESCRIPTION
**Pull request by ilasw**

### CODE REVIEW

File: src/entity/ReputationEvent.ts [hunk 1]
```diff
@@ -23,14 +25,16 @@ export enum ReputationType {
   Source = 'source',
 }
 
-const reputationReasonAmount: Record<ReputationReason, number> = {
+export const reputationReasonAmount: Record<ReputationReason, number> = {
```
Comment (capJavert) [inline src/entity/ReputationEvent.ts:28]: Why are we doing this in separate PR, let's bundle it in the one where it is used? https://github.com/dailydotdev/daily-api/pull/2156#discussion_r1726531910

File: src/entity/ReputationEvent.ts [hunk 2]
```diff
@@ -15,6 +15,8 @@ export enum ReputationReason {
   PostReportConfirmed = 'post_report_confirmed',
   SourceRequestApproved = 'source_request_approved',
   CommentDownvoted = 'comment_downvoted',
+  StreakRecover = 'streak_recover',
+  StreakRecoverForFree = 'streak_recover_for_free',
```
Comment (sshanzel) [inline src/entity/ReputationEvent.ts:19]: I think it might be better to say `StreakFirstRecovery`. Not a blocker though, I think it is more explicit.

Comment (capJavert) [reply src/entity/ReputationEvent.ts:19]: agreed

File: src/entity/ReputationEvent.ts (hunk 1 above)
Comment (ilasw) [reply src/entity/ReputationEvent.ts:28]: Yes, moving to a separate one because it's needed in mutation too

**Review by capJavert — APPROVED** [review]

**Pull request merged**